src/coreclr/vm/methodtable.cpp

```diff
@@ -8911,7 +8912,7 @@ MethodTable::ResolveVirtualStaticMethod(
             // Search for match on a per-level in the type hierarchy
             for (MethodTable* pMT = this; pMT != nullptr; pMT = pMT->GetParentMethodTable())
```

Comment:
Shouldn't this parent type descent also be in a 2-pass loop over <code>allowVariance = FALSE, TRUE</code> when <code>allowVariantMatches</code> is set i.o.w. shouldn't it also be in the below loop used for default interface resolution?

Reply:
Hmm, after getting my head around the tests I take it back, the variant lookup should take place before descending into parent.